layers/+tools/lsp/funcs.el

```diff
@@ -125,7 +124,11 @@
     "x" "text/code"
     "xh" #'lsp-document-highlight
     "xl" #'lsp-lens-show
-    "xL" #'lsp-lens-hide))
+    "xL" #'lsp-lens-hide)
+  (when (configuration-layer/package-used-p 'treemacs)
+    (spacemacs/set-leader-keys-for-minor-mode 'lsp-mode
+      "gh" #'lsp-treemacs-call-hierarchy))
+  )
```

Comment:
the trailing parentheses should not take a new line.

Reply:
done